src/styles/theme/defaultTheme.ts

```diff
@@ -738,12 +738,15 @@ export const defaultTheme = createTheme({
     },
     MuiTabs: {
       styleOverrides: {
-        root: {
+        root: ({ theme }) => ({
           // neglect the default border radius of sibling element (Card component in most cases)
           '&& + *': {
             borderTopLeftRadius: 0,
+            [theme.breakpoints.down('sm')]: {
+              borderTopRightRadius: 0,
```

Comment:
this is quite hacky/unreliable. It's a heuristic that tab names will be longer than tab content on small displays

@donouwens is it okay when it fails like this:
| Before | After |
| --- | --- |
| ![7fd5532d oasis-explorer pages dev_mainnet_emerald_address_0xCB2412a993f406eFf10a74011410a4F35e3549E3 (1)](https://github.com/oasisprotocol/explorer/assets/3758846/c9061ab9-b64e-4e22-bd3c-8f07b1c504a4)  |  ![7fd5532d oasis-explorer pages dev_mainnet_emerald_address_0xCB2412a993f406eFf10a74011410a4F35e3549E3](https://github.com/oasisprotocol/explorer/assets/3758846/57888a8a-c143-4107-928a-e06b23b0ea32) |

Reply:
> this is quite hacky/unreliable. It's a heuristic that tab names will be longer than tab content on small displays

My intention was simply to get rid of rounded corners on the top of the cars under the menu, when on mobile. This is independent from the length of the names. I think on a small screen there is just too much stuff going on near the top of the tab menu; even if we don't see the anomaly (blue/white mixing), I don't consider it a loss that there is no rounded corner there. 

So I don't think it "fails" in the above test case; I think it's a welcome simplification.